<launch>
  <arg name="id" default="0"/>
  <arg name="base_name" default="iRobot_"/>

  <node name="irobotdriver" pkg="irobotcreate2" type="irobotcreate2" respawn="false" output="screen">
    <param name="id_" value="$(arg id)"/>
    <param name="base_name_" value="$(arg base_name)"/>
  </node>
</launch>